packages/platform/platform-mcp/src/utils/toZod.ts

```
@@ -0,0 +1,12 @@
+import {JsonSchema} from "@tsed/schema";
+import {z, type ZodObject} from "zod";
+
+import {jsonSchemaToZod} from "./json-schema-to-zod/index.js";
+
+function transform(schema: JsonSchema): ZodObject {
+  return eval(`(z) => ${jsonSchemaToZod(schema.toJSON(), {zodVersion: 4})}`)(z);
```

Biome (2.3.13)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)